system/security/plugins/htpasswdSecurity/htpasswdSecurity.cpp

```diff
@@ -122,6 +122,12 @@ class CHtpasswdSecurityManager : public CBaseSecurityManager
         return "HTPASSWD Security Manager";
     }
 
+    bool authenticateUser(ISecUser & user, bool &superUser)
+    {
+        superuser = IsPasswordValid(user);//superUser if password is valid
+        return Superuser;
```

Comment:
line 127 sets superuser, but 128 returns Superuser is that the intent?
Beyond that, are all htpassword authenticated users superusers?